### DISCUSSION TITLE
DB

### DISCUSSION THREAD

**Discussion by xitrumcute** [discord-tickets/bot#342] [Support (Q&A)] [answered]
![image](https://user-images.githubusercontent.com/106767375/192144553-cc97f73e-91b2-4b12-9d09-87c2005858ad.png)
What is DB how to add DB

**Comment by RooRay** [comment]
This is not a bug, nor should it be a GitHub issue. 

If you need help use the Discord server or Discussions tab and don't open these garbage issues, please.

**Comment by eartharoid** [comment, marked as answer]
The second line should be either `DB_TYPE=sqlite` or `DB_TYPE=mysql`, not `myqsl=sqlite`.
If you don't have a MySQL database, leave the DB_TYPE as the default and leave the rest to use SQLite automatically.

**Comment by RooRay** [comment]
~~I get an email for each message here I'm gonna linusgun myself~~ nvm fixed

![image](https://user-images.githubusercontent.com/86845749/192150544-8e9e7b8a-7e34-4425-b4a6-7ff474208c42.png)
